packages/elements/src/photon-multirx-form/photon-prescribe-workflow.tsx

```diff
@@ -464,6 +541,10 @@ export function PrescribeWorkflow(props: PrescribeProps) {
                   actions={props.formActions}
                   store={props.formStore}
                   setIsEditing={setIsEditing}
+                  draftedPrescriptionDeleted={function () {
+                    screenDraftedPrescriptions();
```

Comment:
why screen a draft after it's deleted?

Reply:
we'll need to resubmit the screenign request to see who or what has drug interactions or allergens without this draft!